spyderlib/config/main.py

```diff
@@ -580,6 +580,53 @@ def is_ubuntu():
               # ---- Profiler (in p_profiler) ----
               'profiler/run profiler': "F10"
               }),
+            ('emacs_shortcuts',
+              {
+              # -- In codeeditor
+              'editor/code completion': 'Meta+Tab',

```

Comment:
I think we should leave `Ctrl+Space` here because Emacs doesn't have the notion of completion.